app/build.gradle.kts

```diff
@@ -6,11 +6,11 @@ plugins {
 }
 
 android {
-    namespace = "xyz.block.gosling"
+    namespace = "xyz.block.goosemobile"
```

Comment:
is it possible to leave it as gosling (the codename) for paths and namespaces, and just the user facing visible items change?